frontend/src/App.tsx

```diff
@@ -16,8 +17,32 @@ import OpenInNewIcon from "@mui/icons-material/OpenInNew";
 import Typography from "@mui/joy/Typography";
 import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
 import { HEADER_HEIGHT_PX, HEADER_MARGIN_PX } from "./styles/sizes";
+import { Year, LeaderboardEntry } from "./types";
+import { DEFAULT_YEAR } from "./constants";
 
 export default function App() {
+    const [leaderboard, setLeaderboard] = useState<LeaderboardEntry[]>([]);
+    const [leaderboardYear, setLeaderboardYear] = useState<Year>(DEFAULT_YEAR);
+    const [loadingLeaderboard, setLoadingLeaderboard] = useState(false);
+
+    const getLeaderboard = () => {
+        setLoadingLeaderboard(true);
+
+        axios
```

Comment:
Not now, but note for the future - we should configure these calls to read an environment variable (maybe `DEVELOPMENT` or something like that?) and hit `localhost` if it's set. Right now we can only really test against prod :sweat_smile:

Reply:
Agree! Made an issue for it: https://github.com/sirrus233/wotr-community-website/issues/82